### PR TITLE
Hide keystore data

### PR DESCRIPTION
**Pull request by ibencetic-CS**
Moved all keystore data to local.properties.

Example local.properties file:
Screenshot 2023-08-02 at 17 41 59

### CODE REVIEW

File: local.properties.example [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+releaseKeystorePath={path_to_release_keystore}
+releaseKeystorePassword={release_keystore_password}
+releaseKeystoreKeyAlias={release_keystore_key_alias}
+releaseKeystoreKeyPassword={release_keystore_key_password}
+
+messengerKeystorePath={path_to_messenger_keystore}
+messengerKeystorePassword={messenger_keystore_password}
+messengerKeystoreKeyAlias={messenger_keystore_key_alias}
+messengerKeystoreKeyPassword={messenger_keystore_key_password}
```
Comment (MatkoTom) [inline local.properties.example:20]: There is a missing newline at the end of the file, which makes Git angry. Isn't any kind of blocker though.

**Review by MatkoTom — DISMISSED** [review]
LGTM

**Comment by ibencetic-CS** [comment]
@ivolarevic trebat ces local.properties file popuniti s keystore podacima kada pullas ovo

**Review by MatkoTom — APPROVED** [review]
LGTM

**Pull request merged**